src/main/java/org/codehaus/mojo/license/api/DependenciesTool.java

```diff
@@ -117,7 +117,10 @@ public SortedMap<String, MavenProject> loadProjectDependencies( ResolvedProjectD
         SortedMap<String, MavenProject> localCache = new TreeMap<>();
         if ( cache != null )
```

Comment:
NB: Here and further down the `null` check seems redundant, since IIUC callers never pass in `null`. (But we might not want to change this, in order to keep the PR minimal.)